aten/src/ATen/native/hammerblade/Vvadd.cpp

```diff
@@ -11,6 +11,7 @@ Tensor vvadd_hb(const Tensor& self, const Tensor& other) {
 
   // Tutorial TODO:
   // Call HB device kernel tensorlib_vvadd
+  hb_offload_kernel(result, self, other, "tensorlib_vvadd");
```

Comment:
I guess this got into the PR accidentally? We might want to keep vvadd unimplemented for future ppl to work on

Reply:
Oh yes sorry about that, I just removed all the things I had added during the tutorial